embeds.xml

```diff
@@ -11,10 +11,9 @@
     <Include file="Libs\AceGUI-3.0\AceGUI-3.0.xml"/>
     <Include file="Libs\LibSharedMedia-3.0\lib.xml"/>
     <Include file="Libs\AceGUI-3.0-SharedMediaWidgets\widget.xml"/>
-    <Include file="Libs\AceConfigQuestie-3.0\AceConfig-3.0.xml"/>
+    <Include file="Libs\AceConfig-3.0\AceConfig-3.0.xml"/>
     <Include file="Libs\AceComm-3.0\AceComm-3.0.xml"/>
     <Include file="Libs\AceSerializer-3.0\AceSerializer-3.0.xml"/>
-    <Include file="Libs\LibUIDropDownMenuQuestie\LibUIDropDownMenu.xml"/>
```

Comment:
Don't we have to load the Lib on our side to make sure it is available at all (loading just Questie) and in the version we want?

Reply:
Not anymore I don't think, I and our custom fork is really broken with 9.x api

Originally we had to rename a lot because it didnt use LibStub (all globals) the new one is properly contained

Reply:
But if we don't load it at all and no other addon provides the lib then it will not be loaded, will it?

Reply:
Oh nevermind I just realised it is loaded in the TOC file